framework/src/application/schema/constants_schema.js

```diff
@@ -77,9 +77,9 @@ module.exports = {
 			maximum: {
 				$data: '/activeDelegates',
 			},
-			const: 101,
+			const: 10,
 			description:
-				'The maximum number of votes allowed in transaction type(3) votes',
+				'The maximum number of votes allowed in transaction type(13) votes',
```

Comment:
this is removed, so can you add the constant in the HTTP?

Reply:
Was removed by another PR